_includes/homePage.html

```diff
@@ -75,6 +76,15 @@
 	  #integration-container > div:hover{
 		box-shadow: rgb(50 50 93 / 5%) 0px 30px 60px -12px inset, rgb(0 0 0 / 12%) 2px 8px 33px 8px inset;
 	  }
+	  
+	 
+	.company-image img {
+      transition: transform .5s ease-in-out;
+    }
+	.company-image:hover img{
+		transform: scale(1.5);
```

Comment:
Cool idea, @Cvr421, but given that these are non-clickable logos, this visual effect only serves to distract site visitors and cheapens their experience, and consequently, their value perception of the project.

Reply:
ok Lee  i understand that.

Reply:
Thank you